lab4/src/lab4/SearchClass.java

```diff
@@ -0,0 +1,51 @@
+package lab4;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.nio.file.Path;
+import java.util.*;
+
+public class SearchClass {
+
+	public static void main(String[] args) {
+		Scanner scan = new Scanner(System.in); // System.in is a standard input stream.
+
+		System.out.print("Enter search as search <pattern> <file>: ");
+
+		String input = scan.nextLine();
+		scan.close();
+
+		String[] split = input.split(" ");
+		String pattern = split[1];
+
+		File file = new File(System.getProperty("user.dir") + "/src/lab4/" + split[2]);
+	
+			try {
+				Scanner fileScanner = new Scanner(file);
+				boolean present = false;
+				
+				while(fileScanner.hasNextLine()) {
+					String line = fileScanner.nextLine();
+					
+					for(String word : line.split(" ")) {
```

Comment:
Skulle den hitta ord som följs av punkt?

Reply:
We fix!